<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>277</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Game config</string>
  </property>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>42</y>
     <width>35</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimum">
    <number>2</number>
   </property>
   <property name="maximum">
    <number>4</number>
   </property>
  </widget>
  <widget class="QLabel" name="PlayerAmountLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>175</width>
     <height>25</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>Choose the amount of players</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Ok">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Close">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>240</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="player1Name">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>85</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="player2Name">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>120</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="player3Name">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>155</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="player4Name">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>190</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>85</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Player 1 name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="player2NameLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Player 2 name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="player3NameLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>155</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Player 3 name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="player4NameLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Player 4 name:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
